src/stories/List/component.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import StyledList from "./index";
+import StyledListItem from "./ListItem/index";
+import StyledHeader from "../Header";
+
+export default {
+    title: "List",
+    component: StyledList,
+};
+
+export const Default = () => <StyledList>My List</StyledList>;
+
+export const Items = () => (
+    <StyledList>
+        <StyledListItem> Item 1 </StyledListItem>
```

Comment:
Have a look at arrays in more depth 

You can use something like 

const items = [{},{},{}] //array of list item

then
array.map(item => {
  return <StyledListItem {...item}>
})

Reply:
Again hard-coded just to see how it looks first, but .map is definitely the way to go! Done